compiler-rt/test/dfsan/atomic.cpp

```diff
@@ -1,9 +1,12 @@
-// RUN: %clangxx_dfsan %s -fno-exceptions -o %t && %run %t
-// RUN: %clangxx_dfsan -DORIGIN_TRACKING -mllvm -dfsan-track-origins=1 %s -fno-exceptions -o %t && %run %t
+// RUN: %clangxx_dfsan %s -fno-exceptions -D_GLIBCXX_NO_ASSERTIONS -o %t && %run %t
```

Comment:
How is this related to the rest of the change? This doesn't work on aarch64?

Reply:
Looked closer and it's got nothing to do with the architecture, it's because my aarch64 machine has a newer gcc with this change: https://github.com/gcc-mirror/gcc/commit/361d230fd7800a7e749aba8ed020f54f5c26d504

I'll split this into a separate commit.

Reply:
#153873